deployment/local/docker-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+mkdir -p local
+
+if [ -f "local/first_run.lock" ]; then
+    echo "[ + ] Execution of first_run previously completed.  Skipping."
+else
+    echo "[ + ] Executing first_run"
+    python3 install.py --non-interactive --docker
+    touch local/first_run.lock
+fi
```

Comment:
```suggestion

```

Reply:
if statements need to be closed in bash.

govready_q    | /usr/src/app/deployment/local/docker-entrypoint.sh: line 15: syntax error: unexpected end of file